source/neuropod/bindings/c/np_tensor_allocator_internal.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2020 UATC, LLC
```

Comment:
btw, what is UATC, is it Uber ATG abbrevation?

Reply:
`UATC, LLC == Uber ATG`